recipes/flask-injector/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "flask-injector" %}
+{% set version = "0.12.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Flask-Injector-{{ version }}.tar.gz
+  sha256: 82f5bf1245f6fbdbd19eeea4289fe35a3417a8ce05f710847a2f96b771a8085c
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - flask
+    - injector >=0.10.0
+    - python
+    - typing  # [py<35]
```

Comment:
We're not even building packages for Python 3.5 any more. So instead can you add `noarch: python` above under `build:`, and then change the Python dependency here and under `host:` to `python >=3.6`?

Reply:
and go ahead and remove the dependency on `typing`.